perfscale_regression_ci/scripts/network/network_policy_scalability.sh

```diff
@@ -0,0 +1,150 @@
+#/!/bin/bash
+################################################
+## Auth=mifiedle@redhat.com lhorsley@redhat.com
+## Desription: On a 10 node OVN cluster, time how long it takes to scale up to 2000 pods with and without the networkpolicy 
+## Note: The scale up time with the networkpolicy in place will be more than without, but it should not be an order of magnitude difference.	
+## This automated test does not cover step 5.1 - 5.4 in the Polarian test case
+## Polarion test case: OCP-41535 - NetworkPolicy scalability - 2000 pods per namespace using customer network policy	
+## https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-41535
+## Cluster config: Cluster needed in AWS EC2 (m5.xlarge, 3 master/etcd, 3 worker/compute nodes)
+## kube-burner config: perfscale_regerssion_ci/kubeburner-object-templates/node-affinity-anti-affinity-config.yml
+################################################ 
+
+source ../../utils/run_workload.sh
+source ../custom_workload_env.sh
+source ../common.sh
+source network_policy_scalability_env.sh
+
+pod_scale_max=2000
```

Comment:
Let's use the POD_REPLICAS parameter here for our max that we could use to reset in the jenkins UI